core/src/main/scala/org/broadinstitute/dsde/rawls/monitor/workspace/runners/clone/CloneWorkspaceInitStep.scala

```diff
@@ -37,7 +40,23 @@ class CloneWorkspaceInitStep(
     val result = workspaceManagerDAO.getCloneWorkspaceResult(workspaceId, jobId, userCtx)
     result.getJobReport.getStatus match {
       case StatusEnum.SUCCEEDED =>
-        scheduleNextJob(UUID.randomUUID()).map(_ => Complete)
+        val workspaceResult = result.getWorkspace();
+        val errorMessages = new StringBuilder
+        if (workspaceResult != null) {
+          for (resource <- workspaceResult.getResources().asScala)
+            if (resource.getResult() == CloneResourceResult.FAILED) {
+              if (errorMessages.nonEmpty) {
+                errorMessages.append(", ");
```

Comment:
String has a [join](https://docs.oracle.com/javase/8/docs/api/java/lang/String.html#join-java.lang.CharSequence-java.lang.CharSequence...-) method for this sort of concatenation